rpc/handlers_test.go

```diff
@@ -761,6 +811,23 @@ func TestTransactionReceiptByHash(t *testing.T) {
 	block0, err := mainnetGw.BlockByNumber(context.Background(), 0)
 	require.NoError(t, err)
 
+	checkTxReceipt := func(h *felt.Felt, expected string) {
+		t.Helper()
```

Comment:
Not sure if this matters, but I think `t` should be a parameter since we're calling this from subtests?

Reply:
Done